Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/stateselection/FederalStateSelectionViewModel.kt

```diff
@@ -0,0 +1,94 @@
+package de.rki.coronawarnapp.statistics.ui.stateselection
+
+import android.content.Context
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.asLiveData
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+import de.rki.coronawarnapp.datadonation.analytics.common.Districts
+import de.rki.coronawarnapp.datadonation.analytics.common.federalStateShortName
+import de.rki.coronawarnapp.datadonation.analytics.common.labelStringRes
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.PpaData
+import de.rki.coronawarnapp.util.coroutine.DispatcherProvider
+import de.rki.coronawarnapp.util.di.AppContext
+import de.rki.coronawarnapp.util.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.ui.toLazyString
+import de.rki.coronawarnapp.util.ui.toResolvingString
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactory
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+import timber.log.Timber
+
+class FederalStateSelectionViewModel @AssistedInject constructor(
+    @Assisted val selectedFederalStateShortName: String?,
+    @AppContext private val context: Context,
```

Comment:
can you please suppress the warning here ?

Reply:
updated